examples/minimal-tao/src/main.rs

```diff
@@ -27,25 +27,20 @@ fn main() -> Result<(), Error> {
     env_logger::init();
     let event_loop = EventLoop::new();
     let window = {
-        let mut file_menu = MenuBar::new();
-        file_menu.add_native_item(MenuItem::Quit);
-
-        let mut menu = MenuBar::new();
-        menu.add_submenu("File", true, file_menu);
-
         let size = LogicalSize::new(WIDTH as f64, HEIGHT as f64);
-        WindowBuilder::new()
+        let window = WindowBuilder::new()
             .with_title("Hello Pixels/Tao")
-            .with_menu(menu)
```

Comment:
It is sad to see this go, but I know exactly why it had to be: https://github.com/tauri-apps/tao/issues/717#issuecomment-1532055848

I would be inclined to follow up this PR by reintroducing the menu with the `muda` crate.

Reply:
I did try, but it seems like `muda` is a lot less cross-platform than `tao::menu` used to be. For instance, I tried replicating the old menu using `muda::PredefinedMenuItem::Quit`, but that type is unsupported on Linux.

Reply:
No worries, we can look at that separately!